frontend/src/components/LoginForm.vue

```diff
@@ -21,10 +23,12 @@
     <v-text-field 
       class="mb-10 w-100 rounded-lg" 
       variant="outlined"
+      :color="isPasswordInputWithError"
+      :focused="!!isPasswordInputWithError"
```

Comment:
Probably you don't need _!!_. JS should automatically interpolate the variable to be truthy or falsy

Reply:
Leave my variant: :focused can't accept strings